docs/migrations/v7-to-v8.md

```diff
@@ -88,6 +88,39 @@ Each module has a corresponding `MsgUpdateParams` message with a `Params` which
 
 Legacy params subspaces must still be initialised in app.go in order to successfully migrate from x/params to the new self-contained approach. See reference: https://github.com/cosmos/ibc-go/blob/main/testing/simapp/app.go#L1001-L1006
 
+### Governance V1 migration
+
+Proposals have been migrated to [gov v1 messages](https://docs.cosmos.network/v0.50/modules/gov#messages) ref: [#4620](https://github.com/cosmos/ibc-go/pull/4620).
+
+Ensure that the correct authority field is provided to the ibc keeper. The default authority will be `gov` if not specified.
+
+Remove legacy proposal registration from app.go ref: [#4602](https://github.com/cosmos/ibc-go/pull/4602).
+
+Remove the ibcclient ProposalHandler from the govRouter.
+
+```diff
+govRouter := govv1beta1.NewRouter()
+govRouter.AddRoute(govtypes.RouterKey, govv1beta1.ProposalHandler).
+  AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.ParamsKeeper)).
+- AddRoute(ibcclienttypes.RouterKey, ibcclient.NewClientProposalHandler(app.IBCKeeper.ClientKeeper))
+```
+
+Remove the UpgradeProposalHandler from the BasicModuleManager
+
+```diff
+app.BasicModuleManager = module.NewBasicManagerFromManager(
+  app.ModuleManager,
+  map[string]module.AppModuleBasic{
+    genutiltypes.ModuleName: genutil.NewAppModuleBasic(genutiltypes.DefaultMessageValidator),
+    govtypes.ModuleName: gov.NewAppModuleBasic(
+      []govclient.ProposalHandler{
+      paramsclient.ProposalHandler,
+-     ibcclientclient.UpgradeProposalHandler,
```

Comment:
Also the line with `ibcclientclient.UpdateClientProposalHandler,`  should be removed, right?

Reply:
I believe there was one handler that handled both cases